adapter/ipListChecker/adapter.go

```diff
@@ -35,25 +33,27 @@ type adapterState struct{}
 
 func newAdapter() listChecker.Adapter { return &adapterState{} }
 func (a *adapterState) Name() string  { return "istio/ipListChecker" }
+
 func (a *adapterState) Description() string {
 	return "Checks whether an IP address is present in an IP address list."
 }
 
 func (a *adapterState) Close() error { return nil }
 
-func (a *adapterState) ValidateConfig(cfg proto.Message) me.Errors {
+func (a *adapterState) ValidateConfig(cfg proto.Message) error {
```

Comment:
I think it is more natural for ValidateConfig to return ConfigError.

As an adapter author, by simply looking at the API, I will investigate how to construct a config Error and do the correct thing.

There is too much here that is not apparent in the API signature. 
The interaction between multi error / ConfigError has to be studied.